app/models/manageiq/providers/vmware/cloud_manager/vm.rb

```diff
@@ -2,6 +2,8 @@ class ManageIQ::Providers::Vmware::CloudManager::Vm < ManageIQ::Providers::Cloud
   include_concern 'Operations'
 
   supports :snapshots
+  supports :remove_all_snapshots
+  supports_not :remove_snapshots
```

Comment:
```
supports_not :remove_snapshot
```

(without the `s`)

See here for a list of all available features: https://github.com/manageiq/manageiq/blob/master/app/models/mixins/supports_feature_mixin.rb#L61-L139

Reply:
Also, can you please add a spec for the two features above? It's as simple as this: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/spec/models/manageiq/providers/openstack/infra_manager/host_spec.rb#L81-L83